README.md

```diff
@@ -367,6 +369,16 @@ Each of these options can be overridden via command-line options (see [below](#c
 
 A typical configuration for a CI environment will enable all forms of caching, whereas for local development you may want to set `write_remote_cache: false` to avoid waiting for remote cache writes.
 
+### Docker CLI
+
+While still experimental, one change out the CLI command used by Toast. At this time only drop-in replacements for the Docker CLI, such as Podman may be used.
+
+The default values in the configuration file are:
+
+```yaml
+docker_cli: docker        # Allows using other drop-in replacemnts for the Docker CLI such as Podman
```

Comment:
I don't think the indentation before the `#` is necessary since this is separate from the other keys above. It just looks awkward to me?